docs/cloud-tutorials/application-management.md

```diff
@@ -83,6 +83,19 @@ npx dbos-cloud app deploy --previous-version <version-id>
 However, this will fail if the previous and current versions have different database schemas.
 For more information on schema management, see our [schema management guide](./database-management.md#database-schema-management).
 
+### Stored Procedure Versioning
```

Comment:
I think this section belongs to the API reference doc for dbos compiler. Because we don't expect users to explicitly manage stored proc versions, we don't want to expose this complexity to users in this tutorial.

Reply:
I actually had it there originally! :)